README.md

```diff
@@ -1,172 +1,31 @@
-[![Python package](https://github.com//actions/workflows/python-package.yml/badge.svg)](https://github.com//actions/workflows/python-package.yml)
+# Smoothed Particle Hydrodynamics (SPH) via Inductiva API
 
-[![Documentation Status](https://readthedocs.com/projects/inductiva-research-labs-/badge/?version=latest&token=a7a7344c5bc9e511ca715449f94612e4bb7c2dff4c90931778b9c85dbbf5480c)](https://inductiva-research-labs-.readthedocs-hosted.com/en/latest/?badge=latest)
+[**Inductiva API**](https://github.com/inductiva/inductiva/tree/main) empowers users to run large-scale simulations. With a simple interface, users have access to several simulators and features that help manage complex simulation workflows. 
 
-# Start a project
+Here, we explore Smoothed Particle Hydrodynamics (SPH) with two simulators available via **Inductiva API** - [SPlisHSPlasH](https://splishsplash.physics-simulation.org/) and [DualSPHysics](https://dual.sphysics.org/). 
```

Comment:
These links should point back to our SplishSplash calls on the API not to the original solvers.